.gitignore

```diff
@@ -1,3 +1,54 @@
+# These are some examples of commonly ignored file patterns.
```

Comment:
Thanks for adding those back, but please remove this tutorial boilerplate and all the unnecessary extensions from here. We will not have .jar files, .wav files, .exe files, etc etc.

Reply:
Done

Reply:
Still a few that need to be removed, see my comments below.